src/layouts/Footer.astro

```diff
@@ -28,7 +28,7 @@ const footerMenu = await getEntry('menus', 'footer');
         <div class="basis-1/2 flex flex-row flex flex-row gap-2">
           <InlineSVG src="/src/assets/image/flake-white.svg" class="inline-block w-16 h-16" />
           <div class="flex flex-col justify-center">
-            <div>Copyright © 2023 NixOS contributors</div>
+            <div>Copyright © 2024 NixOS contributors</div>
```

Comment:
I guess doing this programatically in the future would be better, how about this?

```suggestion
            <div>Copyright © {new Date().getFullYear()} NixOS contributors</div>
```